nixd/lib/Controller/Completion.cpp

```diff
@@ -386,7 +386,6 @@ void Controller::onCompletion(const CompletionParams &Params,
       const auto AST = CheckDefault(getAST(*TU));
 
       const auto *Desc = AST->descend({Pos, Pos});
-      CheckDefault(Desc && Desc->children().empty());
```

Comment:
Why was this line removed? I believe removing it might lead to null pointer exceptions at line 390.

Reply:
Cant remember, its surely a mistake, will revert !